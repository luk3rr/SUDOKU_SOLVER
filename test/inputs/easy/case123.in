000000090 569100000 200005800 000000060 700020009 081000200 000070300 003580027 402000580 
