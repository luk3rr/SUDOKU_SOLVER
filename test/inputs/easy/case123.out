Solving the following grid:
0 0 0 | 0 0 0 | 0 9 0 
5 6 9 | 1 0 0 | 0 0 0 
2 0 0 | 0 0 5 | 8 0 0 
------+-------+------
0 0 0 | 0 0 0 | 0 6 0 
7 0 0 | 0 2 0 | 0 0 9 
0 8 1 | 0 0 0 | 2 0 0 
------+-------+------
0 0 0 | 0 7 0 | 3 0 0 
0 0 3 | 5 8 0 | 0 2 7 
4 0 2 | 0 0 0 | 5 8 0 


Solution found :')

1 3 8 | 7 4 2 | 6 9 5 
5 6 9 | 1 3 8 | 4 7 2 
2 4 7 | 6 9 5 | 8 3 1 
------+-------+------
3 2 4 | 9 5 1 | 7 6 8 
7 5 6 | 8 2 3 | 1 4 9 
9 8 1 | 4 6 7 | 2 5 3 
------+-------+------
8 9 5 | 2 7 6 | 3 1 4 
6 1 3 | 5 8 4 | 9 2 7 
4 7 2 | 3 1 9 | 5 8 6 

Algorithm: BFS
Total time: 114 ms
Total expanded states: 18812
